integration/ledger/snapshot_test.go

```diff
@@ -1005,3 +1037,68 @@ func invokeWithoutPassingOrdererEndPoint(n *nwo.Network, peer *nwo.Peer, channel
 	invokeChaincode(n, peer, command)
 	nwo.WaitUntilEqualLedgerHeight(n, channelID, nwo.GetLedgerHeight(n, peer, channelID), n.PeersWithChannel(channelID)...)
 }
+
+// commitTx commits a transaction for a given transaction envelope
+func commitTx(n *nwo.Network, orderer *nwo.Orderer, peer *nwo.Peer, channelID string, env *cb.Envelope, txid string) error {
```

Comment:
In this function below, is there a problem if you simply wait for ledger height to increase by one as done in other functions in this file?

Reply:
Got it... then you would have to do more work to get the validation code from block.